examples/yarn.lock

```diff
@@ -1131,6 +1131,13 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@choojs/findup@^0.2.0":
```

Comment:
Also noticed, there is examples/yarn.lock file changed but without package.json changes. Did you installed something? At your branch, if you run yarn in /examples, won't it update yarn.lock again? If not - good, if lockfile changes - you should commit the changed one.

Reply:
Didn't install anything as far as I know. Running yarn and yarn install does not change the yarn.lock file.